aws-rds-optiongroup/src/main/java/software/amazon/rds/optiongroup/Translator.java

```diff
@@ -1,13 +1,8 @@
 package software.amazon.rds.optiongroup;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please don't. `util` is significantly bigger than the range of the classes we're importing here.